### DISCUSSION TITLE
How do I install PyVRP using PyCharm on Windows?

### DISCUSSION THREAD

**Discussion by bmorlo** [PyVRP/PyVRP#576] [Q&A] [answered]
Hey guys,

I'm having a hard time getting started with poetry. Here is where I get stuck:

1) I forked and cloned the repository.
2) ```pip install --upgrade poetry``` worked.
3) Now it looks like I installed poetry in the wrong directory.

P.S. I've been working with PyCharm so far and always installed packages from there (PyPI). Any chance I could work with your code and poetry in PyCharm as well? I found poetry as a package there too.

![image](https://github.com/PyVRP/PyVRP/assets/164156189/c7cbff8a-1295-410a-8a1b-b8aa843bbf51)

**Comment by N-Wouda** [comment, marked as answer]
I do not have the bandwidth to debug installation issues like these. My advice would be to set this up under WSL in a Linux environment, and then connect to that using PyCharm. I think that works, but haven't used PyCharm myself in a few years. In any case, compiling under Windows is quite a hassle so I strongly suggest to use Linux via WSL instead. That might also solve the pathing issue you currently experience.

**Reply by bmorlo** [reply]
Okay, got it! Thank you. Will try WSL
